docs/data/migration/usage-with-material-ui-v5-v6/usage-with-material-ui-v5-v6.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Usage with Material UI v5 and v6
+---
+
+# Usage with Material UI v5/v6
+
+<p class="description">This guide describes the changes needed to use the Data Grid with Material UI v5/v6.</p>
+
+## Package layout changes
+
+MUI X v8 packages have been updated to use the [Node.js `exports` field](https://nodejs.org/api/packages.html#exports), following [Material v7 package layout changes](https://mui.com/system/migration/upgrade-to-v7/#package-layout).
+
+MUI X v8 packages are compatible with Material UI v7 out of the box.
+We encourage upgrading to Material UI v7 to take advantage of better ESM support.
+
+Material UI v6 and v5 can still be used but require some additional steps if you are importing the packages in a Node.js environment.
```

Comment:
Perhaps it makes sense to say "hey, try first without these conditions, if you run into the error e.g. ERR_UNSUPPORTED_DIR_IMPORT, then it means you're trying to load the lib in native Node.js ESM and need to follow this guid"?

Just to prevent users from needlessly setting this up?

Reply:
Done 👍🏻